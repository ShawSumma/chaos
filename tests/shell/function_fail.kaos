void def a()
    print b
end

a()
