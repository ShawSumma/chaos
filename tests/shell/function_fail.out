Chaos Error:
Module: ~/chaos/__interactive__.kaos
Line: 2
Undefined variable: b
Absorbed by Interactive Shell
